Cloud Application Manager/Automating Deployments/running-code-outside-an-instance.md

```diff
@@ -0,0 +1,99 @@
+{{{ "title": "Running Code outside an Instance with Task Variables",
+"date": "26-04-2018",
+"author": "",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+The event scripts are run, by definition, on the instance they belong to. You may have a need, though, to run some code outside the instance's operating system, either because the code must run before the instance has been provisioned or because it has already been terminated. In these cases you may use task variables.
```

Comment:
Please put "task variables" in **bold**

Reply:
I would also add since the beginning that this feature is only available in boxes included in the catalog, and include a link to the doc on how to publish a box.